### PR TITLE
Add new case for set_virtlogd

### PR DESCRIPTION
**Pull request by cliping**
Check virtlogd status after libvirtd is stoped.

Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type q35 conf_file.qemu_conf.set_virtlogd.positive_test.stop_libvirtd --vt-connect-uri qemu:///system
/root/.libvirt-ci-venv-ci-runtest-wk3BTh/lib/python3.6/site-packages/avocado_framework-80.0-py3.6.egg/avocado/plugins/run.py:292: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : 69c710107e23227e2195ab080d14c8c269353fc8
JOB LOG    : /root/avocado/job-results/job-2020-08-05T23.53-69c7101/job.log
 (1/1) type_specific.io-github-autotest-libvirt.conf_file.qemu_conf.set_virtlogd.positive_test.stop_libvirtd: PASS (81.75 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 84.30 s

**Review by Yingshun — APPROVED** [review]

File: libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py [hunk 1]
```diff
@@ -330,14 +331,26 @@ def configure_spice(vm_name):
             cmd = ("lsof  -w |grep pipe|grep virtlogd|tail -n 1|awk '{print %s}'" % pipe_node_field)
             pipe_node = configure(cmd)
 
-            if restart_libvirtd:
-                libvirtd.restart()
+            if restart_libvirtd or stop_libvirtd:
+                cmd2 = "lsof %s | grep virtlogd | awk '{print $2}'" % guest_log_file
+                if restart_libvirtd:
```
Comment (chunfuwen) [inline libvirt/tests/src/conf_file/qemu_conf/set_virtlogd.py:336]: suggest combine this and below operation under same condition ,such as restart_libvirtd or stop_libvirtd

**Comment by chunfuwen** [comment]
@cliping ,if time allows, please consider run this case under split dameon mode, see whether it can work normally too.

**Comment by chloerh** [comment]
> @cliping ,if time allows, please consider run this case under split dameon mode(I assume there are some differences), see whether it can work normally too.

@chunfuwen if there's no other issues, please help merge this pr, it's got 2 approvals already.

**Comment by dzhengfy** [comment]
@chunfuwen , do you need to dismiss your approval because you still have new comments after your approval. Or if you are ok to current implementation, it is ok for you to merge it according to 2 approvals already.

**Pull request merged**